enterprise/frontend/src/embedding-sdk/hooks/private/use-sdk-license-problem.ts

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useMemo, useRef } from "react";
+
+import type { SDKConfig } from "embedding-sdk";
+import { getSdkLicenseProblem } from "embedding-sdk/lib/license-problem";
+import { printLicenseProblemToConsole } from "embedding-sdk/lib/print-license-problem";
+import { useSelector } from "metabase/lib/redux";
+import { getApplicationName } from "metabase/selectors/whitelabel";
+import { getTokenFeature } from "metabase/setup/selectors";
+
+export function useSdkLicenseProblem(config: SDKConfig) {
+  const { allowConsoleLog = true } = config;
+
+  const hasLoggedRef = useRef(false);
+  const appName = useSelector(getApplicationName);
+
+  const hasFeatureFlag = useSelector(state => {
+    // When the settings endpoint has not been called, we assume that the feature is enabled.
+    if (!state.settings.values?.["token-features"]) {
+      return true;
+    }
```

Comment:
I tried to use the existing `useHasTokenFeature` hook, but it throws an error because it is missing this check.